pkg/virtctl/vm/remove_volume_test.go

```diff
@@ -162,4 +173,35 @@ var _ = Describe("Remove volume command", func() {
 		Entry("no persist with dry-run should call VMI endpoint", expectVMIEndpointRemoveVolume(true), "virtualmachineinstances", "--dry-run"),
 		Entry("with persist with dry-run should call VM endpoint", expectVMEndpointRemoveVolume(true), "virtualmachines", "--persist", "--dry-run"),
 	)
+
+	It("should fail immediately on non concurrent error", func() {
+		expectVMIEndpointRemoveVolumeError()
+		commandAndArgs := []string{"removevolume", "testvmi", "--volume-name=testvolume"}
+		cmdRemove := clientcmd.NewRepeatableVirtctlCommand(commandAndArgs...)
+		Expect(cmdRemove()).To(HaveOccurred())
```

Comment:
Assert error that occured more precisely?

Reply:
Sure